src/biophysics.jl

```diff
@@ -370,4 +370,123 @@ function evap(
   J_evap = uconvert(u"g/s",J_evap)
 
   (Q_evap = Q_evap, J_evap = J_evap, J_resp = J_resp, J_cut = J_cut, J_eyes = J_eyes)
+end
+
+function resp(
```

Comment:
```suggestion
function resp(; 
```
This is a lot of positional arguments. We could swap them to being keywords so you have to use the right name but don't need the right order. (just by adding that semi-colon).

Another option is to group some of these in parameter structs at some stage as it becomes clearer what the common groups of evironmental parameters are.

Reply:
`NamedTuple` can also be useful for parameters while we work things out, and can be splatted to keywords. But yes parameter structs are nice for the final design because they can have defaults, and also influence multiple dispatch.